dockerfiles/Dockerfile_1

```diff
@@ -4,7 +4,8 @@ COPY . .
 RUN dotnet publish "web/<changeme>.csproj" -c Release -o /publish
 
 FROM microsoft/dotnet:2.1-aspnetcore-runtime
-ENV TEAM_API_ENDPOINT="http://changeme.westus.cloudapp.azure.com"
+ENV TRIPS_API_ENDPOINT="http://changeme.westus.cloudapp.azure.com" \
```

Comment:
can we change this to something less azure specific. Maybe just "http://endpoint"